### ISSUE TITLE
css broken when having block comment

### ISSUE THREAD

**Issue by eikaramba** [jakubpawlowicz/clean-css#738]
So we are using bootstrap and font awesome and interestingly enough it seems that the following code results in an css, where the .fa class gets removed (it shouldn't)

```
td,th {
  padding: 0;
}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */* {
  box-sizing: border-box;
}

.clearfix:before,.clearfix:after,.dl-horizontal dd:before,.dl-horizontal dd:after,.form-horizontal .form-group:before,.form-horizontal .form-group:after,.btn-toolbar:before,.btn-toolbar:after,.btn-group-vertical > .btn-group:before,.btn-group-vertical > .btn-group:after,.modal-footer:before,.modal-footer:after {
  content: " ";
  display: table;
}

/*!
 *  Font Awesome 4.4.0 by @davegandy - http://fontawesome.io - @fontawesome
 *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
 *//* FONT PATH
 * -------------------------- */.fa {
  display: inline-block;
}
```

Actually you can remove any of the above blocks and then it works. But the actual combination of this css will definitely remove the .fa class. Tested on multiple computers with just

```
cleancss -o test-min.css test.css
```

=>

> td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */*{box-sizing:border-box}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before{content:" ";display:table}

**Comment by jakubpawlowicz** [comment]
It seems `@` characters mess it up. Fix is coming!

**Comment by eikaramba** [comment]
Wuhu great! Then i can relink the stock font-awesome dependency into our build system. Thanks

**Comment by jakubpawlowicz** [comment]
This one was sneaky. Fixed in 3.4.11!
